shared/js/email.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Sends the tracker metrics to the user via Textile pass-through API
+ *
+ * This will be browserifyed and turned into email.js by running 'grunt'
+ */
+
+require.scopes.email = ( () => {
+
+    const ONEDAY = 1000 * 60 * 60 * 24;
+
+    function sendMetrics() {
+
+        chrome.storage.local.get(function (result) {
+
+            var payload = {
+                companyData: result.companyData,
+                totalPages: result.totalPages,
+                totalPagesWithTrackers: result.totalPagesWithTrackers
+            }
+
+            var xhr = new XMLHttpRequest()
+            xhr.open('PUT', 'http://localhost:8000/metrics')
```

Comment:
just hitting my local right now so you'll see some errors in the dev console when the alarm fires